src/parser/DASHTree.cpp

```diff
@@ -468,6 +468,40 @@ start(void *data, const char *el, const char **attr)
             }
             dash->currentNode_ |= DASHTree::MPDNODE_SEGMENTTEMPLATE;
           }
+		  else if (strcmp(el, "ContentProtection") == 0) //Inside Representation
```

Comment:
There are already things in agile branch (protection groups to allow different protection definitions) so it seems for me best that we wait until this is merged back to master to avoid doule work.